ui/src/plugins/dev.perfetto.AndroidLog/logs_track.ts

```diff
@@ -77,7 +77,7 @@ export function createAndroidLogTrack(trace: Trace, uri: string) {
     colorizer: (row) => DEPTH_TO_COLOR[row.depth],
     // It would be nice to show the message on the tooltip too, but loading a
     // message for each event may balloon memory, so we just show the tag.
```

Comment:
Does this apply anymore? Frankly seems a bit excessive concern.

Reply:
This PR doesn't affect this comment in particular. The concern here is that we're loading the message for each event that's loaded into the track into memory, as tooltips need to be synchronously available. Maybe it's not a huge deal though...

Having said that, rendering DOM tooltips does make it easier to format the potentially long message strings with proper wrapping or cropping, which was an issue with the canvas rendering approach.

I'll try it out and upload in a followup if it works out.